client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -91,6 +156,81 @@ class ListAll extends Component {
 		return ! hasAllSitesLoaded || ( requestingFlatDomains && domainsList.length === 0 );
 	}
 
+	isRequestingSiteDomains() {
+		const { requestingSiteDomains } = this.props;
+
+		return (
+			isEmpty( requestingSiteDomains ) ||
+			reduce(
```

Comment:
`reduce` is a standard JS array function; there's no need for lodash here.

Reply:
Yes. 🤦